### ISSUE TITLE
Icon licensing

### ISSUE THREAD

**Issue by solkaz** [elrumo/macOS_Big_Sur_icons_replacements#1042]
I can see that this project uses the [GNU General Public License v3.0](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/blob/master/LICENSE). What is unclear to me is whether the icons themselves (that were uploaded to this project) also fall under the GPL license. This would help maintainers of other projects understand whether they are allowed to use those icons and under what conditions ([example issue](https://github.com/d12frosted/homebrew-emacs-plus/issues/303) in d12frosted/homebrew-emacs-plus).

**Comment by elrumo** [comment]
I'm still trying to figure out how would that work, when I started this the it was very small and I didn't think much about the license at the time. I've been meaning to take off all icons from GitHub for now until I have that figured out.

**Comment by ed9w2in6** [comment]
I would like to know more about this as well since I do want to put it in my personal dotfiles repo. 
Perhaps making sure that when users upload icons they had to agree to release the icons in GPL.

**Comment by texastoland** [comment]
I noticed the Algolia API works well! Would it be worth publishing behind a proxy with your keys there instead of plain text?

**Comment by elrumo** [comment]
@texastoland Yeah, Algolia works amazingly well and surprisingly easy to work with. I've set the keys to work as environmental variables, so that gives it some level of protection. I can't put the search on the backend as that would massively increase the number of requests (now it's almost getting 5M requests per month). But I might eventually migrate to Nuxt and run the search function on the middleware which runs on the server.

**Comment by texastoland** [comment]
I was able to inspect them in the request headers on the site. Just want you to be aware if there's any risk. I was checking because it'd be a nice feature for Unite SSB (not my app).
